assets/js/blocks/products-by-attribute/index.js

```diff
@@ -129,6 +137,10 @@ registerBlockType( blockTypeName, {
 					type: 'number',
 					default: wc_product_block_data.default_rows,
 				},
+				alignButtons: {
```

Comment:
This shouldn't be necessary, the attributes here are a snapshot of what the attributes were when this was deprecated.